test/e2e/getenvoy_extension_test_test.go

```diff
@@ -16,59 +16,57 @@ package e2e_test
 
 import (
 	"os"
-	"path/filepath"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/ginkgo/extensions/table"
-	. "github.com/onsi/gomega"
+	"regexp"
+	"testing"
 
+	"github.com/stretchr/testify/require"
+	"github.com/tetratelabs/getenvoy/pkg/extension/workspace/config/extension"
 	e2e "github.com/tetratelabs/getenvoy/test/e2e/util"
 )
 
-var _ = Describe("getenvoy extension test", func() {
-
-	type testCase e2e.CategoryLanguageTuple
-
-	testCases := func() []TableEntry {
-		testCases := make([]TableEntry, 0)
-		for _, combination := range e2e.GetCategoryLanguageCombinations() {
-			testCases = append(testCases, Entry(combination.String(), testCase(combination)))
-		}
-		return testCases
+// TestGetEnvoyExtensionTest runs the equivalent of "getenvoy extension test" for a matrix of extension.Categories and
```

Comment:
for the doubters.. I've increased the documentation and there are less and more direct output here. I highly doubt someone could say this is a net loss vs what was here before.